### PR TITLE
Route to each entrance of a building

### PR DESCRIPTION
**Pull request by tuukka**

### CODE REVIEW

File: src/planner.ts [hunk 1]
```diff
@@ -61,24 +61,63 @@ export function geometryToGeoJSON(
...
-    .query({
-      from: { latitude: origin[0], longitude: origin[1] },
-      to: { latitude: destination[0], longitude: destination[1] },
-      roadNetworkOnly: true
+  const url = new URL("https://overpass-api.de/api/interpreter");
+  url.searchParams.append(
+    "data",
+    entranceQuery
+      .replace(/LAT/g, destination[0].toString())
+      .replace(/LON/g, destination[1].toString())
```
Comment (haphut) [inline src/planner.ts]: Consider a function instead, something like this:

```js
const buildEntranceQuery = (lat, lon) => {
  return `foo
    ${lat} bar ${lon}
    baz`
}
buildEntranceQuery(lat, lon);
```

File: src/planner.ts [hunk 2]
```diff
@@ -61,24 +61,63 @@ export function geometryToGeoJSON(
...
+  );
+  fetch(url.toString()).then(response =>
+    response.json().then(body => {
```
Comment (haphut) [inline src/planner.ts]: Try existing types or this one from app.quicktype.io:
```ts
export interface OverpassResponse {
    version:   number;
    generator: string;
    osm3s:     Osm3S;
    elements:  Element[];
}

export interface Element {
    type: string;
    id:   number;
    lat:  number;
    lon:  number;
    tags: Tags;
}

export interface Tags {
    entrance:  string;
    operator?: string;
    ref?:      string;
}

export interface Osm3S {
    timestamp_osm_base: Date;
    copyright:          string;
}
```

File: src/planner.ts (hunk 1 above)
Comment (tuukka) [reply src/planner.ts]: Implemented.

File: src/planner.ts (hunk 2 above)
Comment (tuukka) [reply src/planner.ts]: Implemented.

**Pull request merged**